### PR TITLE
ref(dependencies): Update sentry-kafka-schema to 0.1.64

### PR DESCRIPTION
**Pull request by Dav1dde**
Replaces: https://github.com/getsentry/relay/pull/3109

#skip-changelog

### CODE REVIEW

File: relay-kafka/src/producer/schemas.rs [hunk 1]
```diff
@@ -61,42 +52,28 @@ impl Validator {
...
-            }
-
-            return Err(SchemaError::Message(result));
-        }
 
-        Ok(())
+        schema
+            .validate_json(message)
+            .map_err(SchemaError::Validation)
+            .map(drop)
```
Comment (jjbayer) [inline relay-kafka/src/producer/schemas.rs:59]: What is drop?

Comment (jjbayer) [reply relay-kafka/src/producer/schemas.rs:59]: Oh drop!

File: relay-kafka/Cargo.toml [hunk 2]
```diff
@@ -21,15 +21,14 @@ rmp-serde = { version = "1.1.1", optional = true }
 serde = { workspace = true }
 serde_json = { workspace = true, optional = true }
 thiserror = { workspace = true }
-sentry-kafka-schemas = { version = "0.1.12", default_features = false, optional = true }
-jsonschema = { version = "0.17.0", optional = true }
```
Comment (jjbayer) [inline relay-kafka/Cargo.toml:25]: Nice!

**Review by iker-barriocanal — APPROVED** [review]

**Review by jjbayer — APPROVED** [review]

File: relay-kafka/src/producer/schemas.rs (hunk 1 above)
Comment (Dav1dde) [reply relay-kafka/src/producer/schemas.rs:59]: Fancy way of doing `map(|_| ())`

**Pull request merged**